src/typings/RangoAction.ts

```diff
@@ -94,6 +101,7 @@ interface RangoActionWithoutTargetWithOptionalNumberArg {
 }
 
 interface RangoActionWithTargets {
+	arg: any;
```

Comment:
Another weird arg added

Reply:
I was trying to make it accept any sort of Rango action so I thought that we need to have an argument that could work with any rango action. ie be either a distance to scroll or an element or something else like that, 

Now that we have coordinated that we are only going to accept a small subset of actions to script with it probably doesn't make sense to have this any more

Reply:
Although now that I'm looking at it again I'm actually not even sure why this is here. I may have accidentally added this here instead of where the interface should be for the scripting actions

Reply:
Definitely not necessary. The only thing necessary is declaring an interface like this:

```typescript
interface RangoActionRunActionOnReference {
	type: "runActionOnReference";
	arg: RangoActionWithTarget["type"];
	arg2: string;
}
```

The only thing that might be missing here is another optional argument in case we want to scroll relative to a target with a custom scroll amount, but I don't see much use for that